styles.css

```diff
@@ -0,0 +1,99 @@
+body {
+  margin: 0;
+}
+
+/* Navbar */
+.flex-Nav {
+  display: flex;
+  justify-content: space-between;
+  background: #3c3a39;
+  position: absolute;
+  padding: 0 5px 0 5px;
+  top: 0;
+  left: 0;
+  right: 0;
+}
+
+.nav-Button {
+  background-color: #3c3a39;
+  border: 0;
+  width: 14px;
+  flex-basis: 40px;
+  padding: 8px;
+  border-radius: 8px;
+}
+
+.nav-Button:hover {
+  background-color: black;
+}
+
+.header-Name {
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+  line-height: 24px;
+  color: #fff;
+  padding: 10px 12px;
+  margin: 0;
+}
+
+/* headline section */
+#headline {
+  margin: 44px 0 0;
+  height: 590px;
+  display: flex;
+  background: black url(images/Illustration-header.svg) 120px 120px no-repeat;
+  background-size: cover;
+  align-items: center;
+}
+
+#headline .flex-Description {
+  padding: 7%;
+  flex-shrink: 0;
+  display: flex;
+  flex-direction: column;
+  width: 75%;
+}
+
+.headline-Heading {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  margin: 0;
+}
+
+.headline-subHeading {
+  margin: 0;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 32px;
+  color: #fff5e1;
+}
+
+.headline-Text {
+  font-family: 'Poppin', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.headline-Links {
```

Comment:
- [ ] The original design has some different padding and spacing between the icons, you should consider making some changes to match it better.

Figma design:
![image](https://user-images.githubusercontent.com/66967274/143366029-9914aa78-5abf-4a9e-99c7-b69feca91ac5.png)

Your design:
![image](https://user-images.githubusercontent.com/66967274/143366070-3cd7fa86-e372-4454-8d0a-6397b804b3ab.png)

Reply:
Fixed padding between links